src/plugins/ucx/ucx_backend.cpp

```
@@ -133,20 +134,73 @@ class nixlUcxBackendReqH : public nixlBackendReqH {
+    drainRequest(nixlUcxReq req) const {
+        using namespace std::chrono_literals;
+
+        const auto timeout =
+            nixl::config::getValueDefaulted("NIXL_UCX_REQUEST_DRAIN_TIMEOUT", 10'000ms);
+        const auto warning_interval =
+            nixl::config::getValueDefaulted("NIXL_UCX_WARNING_TIMEOUT", 5'000ms);
+        auto next_warning = warning_interval;
+
+        const auto start = std::chrono::steady_clock::now();
+        while (nixl::ucx::ucsToNixlStatus(ucp_request_check_status(req)) == NIXL_IN_PROG) {
+            const auto elapsed = std::chrono::steady_clock::now() - start;
```

Cppcheck (2.21.0)

[style] The function 'getName' is never used.

(unusedFunction)